packages/models-library/src/models_library/api_schemas_catalog/services.py

```diff
@@ -23,23 +23,23 @@
 
 
 class ServiceUpdate(ServiceMetaDataEditable, ServiceAccessRights):
-    class Config:
-        schema_extra: ClassVar[dict[str, Any]] = {
+    model_config = ConfigDict(
+        json_schema_extra={
             "example": {
                 # ServiceAccessRights
                 "accessRights": {
                     1: {
                         "execute_access": False,
                         "write_access": False,
-                    },
+                    },  # type: ignore[dict-item]
```

Comment:
`json_schema_extra: JsonSchemaExtraCallable | JsonDict | None` and `JsonDict` is defined as:

`JsonDict: TypeAlias = Dict[str, JsonValue]`

Reply:
and here we use an int instead of a str. Is that invalid JSON?